src/RacingGame.js

```diff
@@ -0,0 +1,70 @@
+import { inputHandler, outputHandler } from './views/index.js';
+import { GAME_MESSAGES, RULES } from './constants/index.js';
+import {
+  carNameParser,
+  randomNumber,
+  validateCarNames,
+} from './utils/index.js';
+import { Car } from './models/index.js';
+
+class RacingGame {
+  #cars;
+  #tryCount;
+
+  async startGame() {
+    await this.#initialize();
+    await this.#play();
+    await this.#finish();
+  }
+
+  async #initialize() {
+    const carNames = await inputHandler.carNameInput();
+    const parsedNames = carNameParser(carNames);
+    validateCarNames(parsedNames);
+
+    this.#cars = parsedNames.map((name) => new Car(name));
+    this.#tryCount = Number(await inputHandler.racingTryCountInput());
+  }
+
+  async #play() {
+    outputHandler.printMessage(GAME_MESSAGES.RESULT);
+
+    for (let i = 0; i < this.#tryCount; i++) {
+      this.#playRound();
+      outputHandler.printRoundResult(this.#cars);
+      outputHandler.printMessage('');
+    }
+  }
+
+  async #finish() {
+    const winners = this.#getWinners();
+    outputHandler.printMessage(`${GAME_MESSAGES.WINNER}${winners.join(', ')}`);
+  }
+
+  #playRound() {
+    this.#cars.forEach((car) => {
+      const number = randomNumber(
```

Comment:
```suggestion
      const randomNumber = getRandomNumber(
```
변수 명에 `randomNumber`가 오고 함수는 조금 더 동적인 네이밍을 사용하는건 어떨까요??

Reply:
음 그게 나을 것 같아요 ! get을 통해 명확하게 하는게 좋을 것 같네요 !